core/serialization/deserializer/montage-reviver.js

```diff
@@ -753,19 +746,15 @@ var MontageReviver = exports.MontageReviver = Montage.specialize(/** @lends Mont
                 unitDeserializer = new UnitDeserializer(),
                 unitNames;
 
-            try {
-                for (var i = 0, unitsDesc; (unitsDesc = unitsToDeserialize[i]); i++) {
-                    unitNames = unitsDesc.unitNames;
+            for (var i = 0, unitsDesc; (unitsDesc = unitsToDeserialize[i]); i++) {
```

Comment:
how is that backward compatible with default being async?

Reply:
There is no async work being done here, we don't need to wrap the error in a promise.

Reply:
There is no async work being done here, we don't need to wrap the error in a promise.